.test-infra/metrics/grafana/dashboards/perftests_metrics/Python_ML_RunInference_Benchmark_Tests.json

```diff
@@ -424,7 +424,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "alias": "mean_load_model_latency_milli_seconds_resnet101",
+          "alias": "mean_inference_batch_latency_resnet101",
```

Comment:
I actually think I prefer having the milli_seconds piece - its a valuable piece of information for an observer

Reply:
Oh, that makes sense - thanks for clarifying.